eZ Platform/APIs/Languages/README.md

```diff
@@ -0,0 +1,70 @@
+# v2: Updated API Language behaviour
+
+## Objective
+
+Simplify usage of the product for developers, avoid common mistakes when dealing with multi language installs,
+and make sure to cover needs for future UI needs.
+
+### Use Cases
+The identified new behaviour:
+- As a Developer, I want to get API entities in prioritised language of the given SiteAccess,
+  so I don't need to specify language in API Service calls
+- As a Developer, I want to get API entities in one Translation at a time,
+  so I don't need to specify language or use helpers when using the API entities
+
+##### Related use cases
+- As an Editor, in UI I want to be able to select languages of API entities to view and edit translation independently
+  of the current SiteAccess, _so I don't need to deal with several variants of backend to do that_
```

Comment:
> note: the 2nd use case is a no brainer for content, but I am unsure if it needs to be implemented for all kind of entities.

It would be a pity to not be fully able to translate everything from UI, we have all the pieces for it now, and we know we'll meet this requriment as the reason why we have this in the first place is that there were in the past for legacy.